src/axolotl/utils/data.py

```diff
@@ -336,6 +336,16 @@ def for_d_in_datasets(dataset_configs):
                         split=None,
                         storage_options=storage_options,
                     )
+            elif config_dataset.path.startswith("http"):
```

Comment:
do we want to allow loading from non-ssl urls?

Reply:
ok I'll add the `s`!